src/rtctools_heat_network/workflows/grow_workflow.py

```diff
@@ -780,19 +426,6 @@ def heat_network_options(self):
 
```

Comment:
I think it is better to always set minimum velocity to zero, just to avoid unexpected cases where we cannot match demand.

Reply:
updated